### DISCUSSION TITLE
part-ii/2021-130/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#87] [Question comments]
Part II 2021 Quantum Information and Computation: Paper 3, Section II, D 

https://questions.tripos.org/part-ii/2021-130/

**Comment by thenewtonium** [comment]
For describing the geometry in **(a)** it helps to fix directions for describing rotations, e.g. ‘Define *clockwise* to be the direction from $\ket{x_0}$ to $\ket{\psi_0}$ in the plane‘.
